flocker_bb/ec2_buildslave.py

```diff
@@ -65,31 +65,35 @@
 from buildbot.buildslave.base import BuildSlave
 from buildbot import config
 
-from flocker_bb.ec2 import EC2
 from machinist import WrongState
 
 
-class EC2BuildSlave(BuildSlave):
+class OnDemandBuildSlave(BuildSlave):
```

Comment:
Based on this class being renamed and generalized, we want to rename this module too.  We decided not to rename it before the PR was reviewed in case the renaming screwed up the diff too badly.